autoPyTorch/datasets/resampling_strategy.py

```diff
@@ -225,3 +243,30 @@ def get_cross_validators(cls, *cross_val_types: CrossValTypes) -> Dict[str, Cros
             for cross_val_type in cross_val_types
         }
         return cross_validators
+
+
+class NoResamplingFuncs():
```

Comment:
I forget again, but when do we use `NoResamplingStrategy`?

Reply:
We use it when we don't want to split the training data. For example, this `TestEvaluator` returns the test loss as the cost. This will be used for portfolio construction as described in the autosklearn 2 paper. Another use case, use it to fit the incumbent configuration using all of the data. This was used to provide the final accuracy in the 'Regularisation Cocktails' paper.

Reply:
Ah, I got it.
Basically, this is used at the very final training.